storage/diskmetricstore.go

```diff
@@ -134,6 +137,24 @@ func (dms *DiskMetricStore) Shutdown() error {
 	return <-dms.done
 }
 
+func (dms *DiskMetricStore) Healthy() (bool, error) {
```

Comment:
Need a doc comment, something like `// Healthy implements the MetricStore interface.` will suffice.

Reply:
done